consent.yaml

```diff
@@ -185,6 +185,260 @@ paths:
                 $ref: '#/components/schemas/ConsentResponse'
         429:
           $ref: '#/components/responses/429RateLimited'
+  /consents/v2:
+    get:
+      security:
+        - bearerAuth:
+            - 'consents:read'
+        - m2m:
+            - 'consents:read'
+      tags:
+        - Consent
+      summary: Get Consents
+      description: Get all consent records for this consumer
+      operationId: findAllConsentsV2
+      parameters:
+        - name: consumerId
+          in: query
+          description: 'Please ensure proper encoding of this parameter as identifier values from some IDPs could have special characters like ''|'', which need be encoded as ''%7c''. <br/> consumerId should only be used with machine token, otherwise it will be deemed BAD_REQUEST'
+          example: auth0%7c5fbc585628421a006e83d95d
+          required: false
+          schema:
+            type: string
+        - name: consumerEmail
+          in: query
+          example: hello@adatree.com.au
+          required: false
+          schema:
+            type: string
+        - name: consentId
+          in: query
+          example: 148c9187-e52b-431f-9b24-7225e75ca2be
+          required: false
+          schema:
+            type: string
```

Comment:
this one too https://github.com/Adatree/api-specs/pull/47/files#diff-fa21ef9cc73e0385ed1e3d52420c3d0871b0b4880678a5e40eebec4d18b11543R1685-R1692